### PR TITLE
Fix console errors

### PR DESCRIPTION
**Pull request by mirestrepo**

### CODE REVIEW

**Review by eldu — COMMENTED** [review]
Looks good to me ! 

Sort of not related to this PR, when I go to review changes in this PR, it looks like 
[there are some style warnings](https://user-images.githubusercontent.com/5270855/145619202-961b4d67-9ed4-4231-9610-fa71cec2a676.png). In provident we have `"lint": "vue-cli-service lint",` as well as a githook to lint vue. I noticed you did have a linter for js, just wondering if you'd be interested in a lint for vue as well ! 

Also looks like ci checks have failed?

File: components/basejob/Basejobfragment.vue [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
-    <fragment v-if="sbatch[sbatchIndex].memory">#SBATCH --mem {{ sbatch[sbatchIndex].memory }} </fragment>
-    <fragment v-if="sbatch[sbatchIndex].time">#SBATCH --time {{ timeString }} </fragment>
+    <fragment >#SBATCH --partition {{ sbatch[sbatchIndex].partition }}</fragment>
+    <fragment >#SBATCH --nodes {{ sbatch[sbatchIndex].nnodes }} </fragment>
+    <fragment >#SBATCH --cpus-per-task {{ sbatch[sbatchIndex].ncpus }} </fragment>
+    <fragment >#SBATCH --mem {{ sbatch[sbatchIndex].memory }} </fragment>
+    <fragment >#SBATCH --time {{ timeString }} </fragment>
     <fragment v-if="sbatch[sbatchIndex].output">#SBATCH --output {{ sbatch[sbatchIndex].output }} </fragment>
     <fragment v-if="sbatch[sbatchIndex].emailevents.length > 0">
     #SBATCH --mail-type {{ emailEventString }}
```
Comment (eldu) [inline components/basejob/Basejobfragment.vue:13]: For these v-ifs that were removed, do you want this to show all the time? Just checking!

**Review by eldu — APPROVED** [review]
Approved! Looks really good! 

I did throw in a comment question though !

File: components/basejob/Basejobfragment.vue (hunk 1 above)
Comment (eldu) [reply components/basejob/Basejobfragment.vue:13]: Could have a wider parent div like `<div v-if="sbatch[sbatchIndex]></div>`

Comment (mirestrepo) [reply components/basejob/Basejobfragment.vue:13]: Great question! I realized that these inputs always have a value. So if I want to make them optional I need to think of a way to have a different validation or UI component to disable. So yes, this is intentional for now!

**Pull request merged**